pkg/downloads/versions.go

```diff
@@ -2,7 +2,7 @@
 // or more contributor license agreements. Licensed under the Elastic License;
 // you may not use this file except in compliance with the Elastic License.
 
-package internal
+package downloads
```

Comment:
@adam-stokes I could not find a better name for this package. Maybe you have something better in mind